hadoop-ozone/dist/src/main/smoketest/s3/freon.robot

```diff
@@ -41,4 +43,4 @@ Freon S3BG
 *** Test Cases ***
 Run Freon S3BG
     [Setup]    Setup aws credentials
-    Freon S3BG
\ No newline at end of file
+    Freon S3BG   s3bg-${BUCKET}
```

Comment:
Hi @adoroszlai,  not understanding here what is the extra arg ```s3bg-${BUCKET}``` used for.
Seems the s3bg -p ${prefix} appointed the fixed prefix which is ```s3bg```

Reply:
Robot keyword arguments can be specified as positional or named.  So `s3bg-${BUCKET}` is for the first parameter, `prefix`.

```
ozone-ec.xml
2907:<msg timestamp="20230912 19:41:21.443" level="INFO">Running command 'ozone freon s3bg -e http://s3g:9878 -t 10 -n 100 -p s3bg-erasure  2&gt;&amp;1'.</msg>

ozone-ha.xml
14246:<msg timestamp="20230912 19:53:20.319" level="INFO">Running command 'ozone freon s3bg -e http://s3g:9878 -t 10 -n 100 -p s3bg-bucket-ozone-test-2251842976  2&gt;&amp;1'.</msg>

ozone-s3-haproxy.xml
2921:<msg timestamp="20230912 19:55:39.308" level="INFO">Running command 'ozone freon s3bg -e http://s3g:9878 -t 10 -n 100 -p s3bg-bucket-ozone-test-7696293952  2&gt;&amp;1'.</msg>

ozonesecure-ha.xml
21939:<msg timestamp="20230912 20:04:05.210" level="INFO">Running command 'ozone freon s3bg -e http://s3g:9878 -t 10 -n 100 -p s3bg-encrypted  2&gt;&amp;1'.</msg>

ozonesecure-vault.xml
3364:<msg timestamp="20230912 19:45:28.638" level="INFO">Running command 'ozone freon s3bg -e http://s3g:9878 -t 10 -n 100 -p s3bg-bucket-ozone-test-9555341792  2&gt;&amp;1'.</msg>
```